package/lib/src/beamer_delegate.dart

```diff
@@ -298,7 +298,7 @@ class BeamerDelegate extends RouterDelegate<RouteInformation>
   /// [beamParameters] hold various parameters such as `transitionDelegate` and
   /// other that will be used for this update **and** stored in history.
   ///
-  /// [data] can be used to hold arbitrary [Object] througout navigation.
+  /// [data] can be used to hold arbitrary [Object] through navigation.
```

Comment:
It was a typo :sweat_smile:, but I would like to keep the word `throughout`. It describes it more precisely than `through`

Reply:
Oh, my bad. Fixed.

Reply:
Oh, my bad. Fixed.